src/WorkItemTracking/WorkItemTrackingServices.ts

```diff
@@ -135,6 +135,20 @@ export interface IWorkItemFormService {
     * @returns {Promise<{ [fieldName: string]: boolean }>} A promise that returns a dictionary of field value update results (refName to results pairs).
     */
     setFieldValues(fields: { [fieldName: string]: Object }): Promise<{ [fieldName: string]: boolean }>;
+    /**
+     * Filters the allowed values for the field on the active work item. This could only be applied to the fields with a picklist. 
+     * 
+     * @param {string} fieldReferenceName Field reference name.
+     * @param {string[]} values Filtered field values. Values should be a subset of the field's allowed values.
+     */
+    filterAllowedFieldValues(fieldReferenceName: string, values: string[]): Promise<void>;
+    /**
+     * Gets only filtered allowed values for the field on the active work item.
+     * 
+     * @param {string} fieldReferenceName Field reference name.
+     * @returns {Promise<Object[]>} A promise that returns array of filtered allowed values.
+     */
+    getFilteredAllowedFieldValues(fieldReferenceName: string): Promise<Object[]>;
```

Comment:
Should the return type be Promise<string[]> ?

Reply:
Since filterAllowedFieldValues also takes in a string array

Reply:
Yes, it is better to be Promise<string[]>. I will update PR.